docs/data/material/getting-started/supported-platforms/supported-platforms.md

```diff
@@ -39,7 +39,7 @@ v6 will completely remove the support of IE 11.
 
 <!-- #stable-snapshot -->
 
-MUI supports [Node.js](https://github.com/nodejs/node) starting with version 12.17 (or 12.0 with `--experimental-modules` enabled) for server-side rendering.
+MUI supports [Node.js](https://github.com/nodejs/node) starting with version 12.0 for server-side rendering.
```

Comment:
MUI or Material UI?

Reply:
I think that we can envision https://mui.com/material-ui/getting-started/supported-platforms/ as the shared guidelines for all the products, so I would suggest we keep MUI.